toolkit/javascripts/analytics/_init.js

```diff
@@ -0,0 +1,10 @@
+(function(root) {
+  "use strict";
+  // We've copied this from the Briefs FE
+  root.GOVUK.GDM.analytics.init = function () {
+    this.register();
```

Comment:
There's something wrong here about the layout, although I get that we're just moving the existing thang... might it make sense to have the `// include` things here, so that you only have to include `_init.js` and then you get the rest? (In fact, at that point maybe you should lose the underscore - we should be finding a way to indicate _something_ as the entry point for this code?)

Reply:
Yep, that makes sense to me.

Reply:
I'm tempted to keep the underscore though, as the FE apps have everything underscored apart from the file we actually end up serving (`application.js`)